e2e/core/datatable/data-table-component.e2e.ts

```diff
@@ -180,7 +180,8 @@ describe('Datatable component', () => {
             await expect(await dataTablePage.getClipboardInputText()).toEqual('1');
         });
 
-        it('[C307101] A column value of type json and with copyContent set to true is copied when clicking on it', async () => {
+        // tslint:disable-next-line: ban
+        xit('[C307101] A column value of type json and with copyContent set to true is copied when clicking on it', async () => {
```

Comment:
one test skipped..

Reply:
yeah, I need to remove that test :) we no longer have that scenario (was before the button)